pkgs/development/libraries/boost/generic.nix

```diff
@@ -174,6 +174,10 @@ stdenv.mkDerivation {
     inherit boostBuildPatches;
   };
 
+  postPatch = ''
+    patchShebangs tools/build/src/engine/build.sh
+  '';
```

Comment:
This should only apply when `lib.versionAtLeast version "1.82"` is true.

Reply:
I was wondering whether that would be necessary or not, now I think I understand that without the version check there are a lot of rebuilds, correct?

Reply:
Yes. In the meantime, I think that `postPatch = ""` will still cause rebuilds, so maybe `postPatch = if cond then expr else null` instead.

Reply:
Thanks for the helpful info, @wegank. I've verified that on my machine `boost181` isn't rebuild. 👍